### PR TITLE
Generate cc and header files for hello world model.

### PR DESCRIPTION
**Pull request by njeffrie**
Moving all examples to use test data generation for models and test inputs.

BUG=b/196065539

### CODE REVIEW

**Review by advaitjain — COMMENTED** [review]
Can we remove model.cc and model.h from examples/hello_world?

File: tensorflow/lite/micro/examples/hello_world/Makefile.inc [hunk 1]
```diff
@@ -18,30 +14,41 @@ tensorflow/lite/micro/examples/$(EXAMPLE_NAME)/constants.h
...
-tensorflow/lite/micro/examples/$(EXAMPLE_NAME)/model.h \
 tensorflow/lite/micro/examples/$(EXAMPLE_NAME)/output_handler.h \
 tensorflow/lite/micro/examples/$(EXAMPLE_NAME)/constants.h \
 tensorflow/lite/micro/examples/$(EXAMPLE_NAME)/main_functions.h
 
+HELLO_WORLD_GENERATOR_INPUTS := \
+tensorflow/lite/micro/examples/hello_world/hello_world.tflite
+
+HELLO_WORLD_GENERATED_SRCS := \
+tensorflow/lite/micro/models/hello_world_model_data.cc
```
Comment (advaitjain) [inline tensorflow/lite/micro/examples/hello_world/Makefile.inc:29]: update?

File: tensorflow/lite/micro/examples/hello_world/Makefile.inc [hunk 2]
```diff
@@ -18,30 +14,41 @@ tensorflow/lite/micro/examples/$(EXAMPLE_NAME)/constants.h
...
+
+HELLO_WORLD_GENERATED_HDRS := \
+tensorflow/lite/micro/models/hello_world_model_data.h
```
Comment (advaitjain) [inline tensorflow/lite/micro/examples/hello_world/Makefile.inc:32]: update?

**Review by advaitjain — APPROVED** [review]

File: tensorflow/lite/micro/examples/hello_world/Makefile.inc (hunk 2 above)
Comment (njeffrie) [reply tensorflow/lite/micro/examples/hello_world/Makefile.inc]: Done.

File: tensorflow/lite/micro/examples/hello_world/Makefile.inc (hunk 1 above)
Comment (njeffrie) [reply tensorflow/lite/micro/examples/hello_world/Makefile.inc]: Done.

**Review by advaitjain — APPROVED** [review]

**Pull request merged**